ugvc/scripts/sorter_to_h5.py

```diff
@@ -63,51 +63,56 @@ def sorter_to_h5(
             h5_dict[row["H5_key"]][row["H5_item"]] = json_data[row["key"]]
 
     # extract coverage stats
-    df_coverage_histogram = (
-        pd.concat(
-            (pd.DataFrame(json_data["base_coverage"][key], columns=[key]) for key in json_data["base_coverage"].keys()),
-            axis=1,
+    if not json_data["base_coverage"] == {}:  # if base_coverage is not empty
```

Comment:
To be on the safe side I would do
`if "base_coverage" in json_data and len(json_data["base_coverage"]) == 0`

I wouldn't rely on the fact that Ohad creates an empty field